README.md

```diff
@@ -1,70 +1,55 @@
-# Getting Started with Create React App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+#  Covid app
 
-## Available Scripts
+> This project was built using react redux to fetch 
+> covid and country's details from exteranal APIs
 
-In the project directory, you can run:
+![screenshot](./image.png)
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Built With
 
-### `npm test`
+- React 
+- React redux
+- React test libraries
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Link 
+[LINK LIVE](https://coviid.netlify.app/)
+[LOOM LINK](https://www.loom.com/share/a5ffa81c46d64a61baa132ecad7dfa99)
+## Getting Started
 
-### `npm run build`
+**To get a copy of this repo into your local machine run the following command:**
+```
+git clone  git@github.com:aakbarkhan/metrics-webapp.git
+cd matrices-app
+in visual studio code you can click the go live button to view it live in the browser.
+```
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Authors
```

Comment:
- [ ] Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.